internal/app/server/server.go

```diff
@@ -63,7 +88,8 @@ func RunWithContext(ctx context.Context, cancel context.CancelFunc, bootstrapCon
 	requestMapper := mapper.NewMapper(aggregationRulesConfig)
 
 	// Initialize orchestrator.
-	orchestrator := orchestrator.New(ctx, logger, requestMapper, upstreamClient, bootstrapConfig.Cache)
+	orchestrator := orchestrator.New(ctx, logger, scope.SubScope("orchestrator"), requestMapper,
```

Comment:
possibly nit, but should scope.SubScope("orchestrator") be declared a const along with metricSubscope?